tests/model_explainability/guardrails/utils.py

```diff
@@ -0,0 +1,188 @@
+import http
+
+from requests import Response
+from simple_logger.logger import get_logger
+from typing import Dict, Any, List, Optional
+
+LOGGER = get_logger(name=__name__)
+
+
+def get_auth_headers(token: str) -> Dict[str, str]:
+    return {"Content-Type": "application/json", "Authorization": f"Bearer {token}"}
+
+
+def get_chat_payload(content: str) -> Dict[str, Any]:
+    return {
+        "model": "/mnt/models",
+        "messages": [
+            {"role": "user", "content": content},
+        ],
+    }
+
+
+def verify_and_parse_response(response: Response) -> Dict[str, Any]:
+    if response.status_code != http.HTTPStatus.OK:
+        assert False, f"Expected status code {http.HTTPStatus.OK}, got {response.status_code}"
+
+    try:
+        return response.json()
+    except ValueError:
```

Comment:
Would not it raise `requests.exceptions.JSONDecodeError` https://requests.readthedocs.io/en/latest/user/quickstart/#json-response-content